.vscode/launch.json

```
@@ -15,6 +15,17 @@
         "DEBUGTELEMETRY": "v"
       }
     },
+    {
+      "name": "Build, Run and Debug Data Mapper Extension In Dev Mode",
```

Is this new? I haven't seen this before.

```suggestion
      "name": "Build, Run, and Debug Data Mapper Extension in Dev Mode",
```